opencog/cogserver/scm/config.scm

```diff
@@ -44,6 +44,7 @@
 (add-to-ltdl-path (string-append (getcwd) "/opencog/nlp/sureal"))
 (add-to-ltdl-path (string-append (getcwd) "/opencog/nlp/lg-dict"))
 (add-to-ltdl-path (string-append (getcwd) "/opencog/planning"))
+(add-to-ltdl-path (string-append (getcwd) "/opencog/scm/opencog/learning"))
```

Comment:
Will removed this line... I added it when I was trying to fix the "lib not found" problem, and forgot to removed it when I pushed.